big-red-shuttle/Location.swift

```diff
@@ -45,6 +50,9 @@ class Location: NSObject, CLLocationManagerDelegate {
             print("longitude: \(location.longitude), latitude: \(location.latitude)")
             currentUserLocation = Coordinate(location: location, timestamp: Date())
         }
+        
+        fetchedUserLocationCompletionBlock?()
+        fetchedUserLocationCompletionBlock = nil
```

Comment:
temporary hack until i figure our how to wait for location data to come in before we start logging